sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/LiteralExpressionSuite.scala

```diff
@@ -200,8 +204,8 @@ class LiteralExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkSeqLiteral(Seq(1, 2, 3), IntegerType)
     checkSeqLiteral(Seq("a", "b", "c"), StringType)
     checkSeqLiteral(Seq(1.0, 4.0), DoubleType)
-    checkSeqLiteral(Seq(MICROS_PER_DAY, MICROS_PER_HOUR),
-      CalendarIntervalType)
```

Comment:
Could you mention this removal (replacement) in the PR description, @MaxGekk ?

Reply:
@maropu I removed it because this doesn't check `CalendarIntervalType`. `elementType` is not used in `checkSeqLiteral()`, and this `checkSeqLiteral(Seq(MICROS_PER_DAY, MICROS_PER_HOUR), CalendarIntervalType)` checks the `Long` type, actually.

I think of to improve the test by removing `elementType` at all, and check `CalendarIntervalType` via `CalendarInterval` values.

Reply:
done